code/LeftAndMain.php

```diff
@@ -115,6 +115,12 @@ class LeftAndMain extends Controller implements PermissionProvider
      */
     private static $url_priority = 50;
 
+    /**
+     * @config
+     * @var string
+     */
+    private static bool $ignore_menuitem = false;
```

Comment:
```suggestion
     */
    private static bool $ignore_menuitem = false;
```
`string` is obviously not the correct var type - but we don't need any `@var` in the PHPDoc since this has a strong typehint anyway.

`@config` doesn't actually do anything, so removing that too.

Please also add a PHPDoc description of what this configuration does.

Reply:
Done, thanks for the input